MULTIPLATFORM.md

```diff
@@ -0,0 +1,244 @@
+# Kotlin/Native in multiplatform projects
+
+*TODO: Some introduction*
+
+## Creating multiplatform Android/iOS application with Kotlin
+
+*TODO: High-level application structure: what code is common (data, logic) and what code is platform (io, ui, network).
```

Comment:
To create MPP application one has start with clear understanding which parts of an application is common for a different targets, and which ones are specific, and organize module structure accordingly.  For shared Kotlin code the common ground consist of the Kotlin's standard library, which does include basic data structures and computational primitives, along with expect classes with platform-specific implementation. Most frequently, such code consists of GUI, input-output, cryptography and other APIs, available on the particular platform.

Reply:
Done